test/type-applications/Main.hs

```diff
@@ -368,3 +528,14 @@ withRenamedVals ::
 withRenamedVals vals f = case runRenameM . traverse renameValT $ vals of
   Left err -> counterexample (show err) False
   Right vals' -> f vals'
+
+_withRenamedDatadecl ::
+  DataDeclaration AbstractTy ->
+  (DataDeclaration Renamed -> Property) ->
+  Property
+_withRenamedDatadecl decl f = case runRenameM . renameDataDecl $ decl of
+  Left err -> counterexample (show err) False
+  Right decl' -> f decl'
```

Comment:
Is this still in use? If so, and it's a helper, it should be named without the underscore; if not, remove it.